e2e/http_funcs.go

```diff
@@ -14,7 +14,7 @@ func GetBody(rawURL string, cookies ...*http.Cookie) (string, error) {
 	for _, c := range cookies {
 		req.AddCookie(c)
 	}
-	resp, err := http.DefaultClient.Do(req) //nolint[:gosec]
+	resp, err := http.DefaultClient.Do(req) //nolint:gosec //reason test code
```

Comment:
`//nolint[:gosec]` is a wrong syntax (most likely from previous linter we had in place). With this syntax **all** linter checks are bypassed. The proper syntax is `//nolint:linter1,linter2 //reason explanation`. This PR fixes it for all existing ones.